administrator/components/com_media/Controller/Api.php

```diff
@@ -104,7 +105,20 @@ public function files()
 			switch ($method)
 			{
 				case 'get':
-					$data = $this->getModel()->getFiles($adapter, $path, $this->input->getWord('filter'));
+					$option = $this->input->getWord('opt');
```

Comment:
I would rather make it as an option that the get response should include the permalink or not in the returned data. IMO the GET response should always return the same data, but with some more details or not.